Sources/TensorFlow/Loss.swift

```diff
@@ -287,5 +287,8 @@ public func sigmoidCrossEntropy<Scalar: TensorFlowFloatingPoint>(
 ) -> Tensor<Scalar> {
     // This numerically stable implementation is based on the TensorFlow Python API.
     let maxLogitsWithZero = max(logits, Tensor(0))
-    return reduction(maxLogitsWithZero - logits * labels + log(1 + exp(-abs(logits))))
+    // Note: `result` is split into two lines to avoid the "compiler is unable to type-check this
```

Comment:
@dan-zheng this part.

Reply:
Yes, it was weirdly necessary to fix compilation, though I'm not sure why. [This comment](https://github.com/apple/swift/pull/26527#issuecomment-518935206) provides a bit more context.

Compilation fixes are necessary 🙂

Reply:
@dan-zheng Ah, Alright thanks for the clarification :)